include/pttstruct.h

```diff
@@ -284,23 +284,24 @@ typedef struct fileheader_t { /* 128 bytes */
 	} refer;
     }	    multi;		    /* rocker: if bit32 on ==> reference */
     /* XXX dirty, split into flag and money if money of each file is less than 16bit? */
-    unsigned char   filemode;        /* must be last field @ boards.c */
-    char    pad3[3];
+    uint16_t   filemode;        /* must be last field @ boards.c */
```

Comment:
Note this is assuming little endian. @wens

Reply:
Yes we can assume little endian. There aren't many big endian Linux distro ports anymore, at least not for modern hardware. I doubt the code base was ever tested on big endian either. It might be full of bugs lol.

Reply:
Don't forget somebody have to clear the pad to zero before deploy this code.